bedrock/base/templates/includes/protocol/footer/footer-refresh.html

```diff
@@ -50,6 +50,7 @@ <h2 class="moz24-footer-label" data-testid="footer-heading-support">
               <li><a href="https://support.mozilla.org/?{{ utm_params }}&utm_content=support" data-link-position="footer" data-link-text="Product Help">{{ ftl('footer-refresh-product-help', fallback='footer-refresh-support') }}</a></li>
               <li><a href="https://bugzilla.mozilla.org/?{{ utm_params }}&utm_content=support" data-link-position="footer" data-link-text="File a Bug">{{ ftl('footer-refresh-file-a-bug') }}</a></li>
               <li><a href="https://pontoon.mozilla.org/?{{ utm_params }}&utm_content=support" data-link-position="footer" data-link-text="Localise Mozilla">{{ ftl('footer-refresh-localize-mozilla') }}</a></li>
+              <li><a href="https://www.mozilla.org/security/" data-link-position="footer" data-link-text="Security">{{ ftl('footer-refresh-security') }}</a></li>
```

Comment:
Is there a urlpattern for this path? Probably not named explicitly, but the app would expose that most likely as `url('security')` or `url('security.index')` …?

```suggestion
              <li><a href="{{ url('security') }}" data-link-position="footer" data-link-text="Security">{{ ftl('footer-refresh-security') }}</a></li>
```